### PR TITLE
Add workspace query worker

### PR DESCRIPTION
**Pull request by Tulean**
JIRA: RAIL-2902

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/api-client-tiger/src/axios.ts [hunk 1]
```diff
@@ -22,6 +22,9 @@ const _CONFIG: AxiosRequestConfig = {
         put: {
             "Content-Type": "application/json;charset=utf8",
         },
+        get: {
+            Accept: "application/vnd.gooddata.api+json",
```
Comment (no23reason) [inline libs/api-client-tiger/src/axios.ts]: Is this safe? Can we instead use the default headers specified in https://github.com/gooddata/gooddata-ui-sdk/pull/970/files#diff-72216b0b9af677699864dac77f0165de1c8efd82a99101f3eaf8d23c5f6fddc7R84

Comment (no23reason) [reply libs/api-client-tiger/src/axios.ts]: I mean changing the default for all the get requests can yield unexpected results.

Comment (Tulean) [reply libs/api-client-tiger/src/axios.ts]: makes sense, changed it

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by michalvanco** [comment]
Closing, commit is applied in this PR: https://github.com/gooddata/gooddata-ui-sdk/pull/970

**Pull request closed without merging**